db/channel_cache.go

```diff
@@ -231,6 +231,8 @@ func (c *channelCacheImpl) GetCachedChanges(channelName string) []*LogEntry {
 // CleanAgedItems prunes the caches based on age of items. Error returned to fulfill BackgroundTaskFunc signature.
 func (c *channelCacheImpl) cleanAgedItems(ctx context.Context) error {
 
+	base.InfofCtx(ctx, base.KeyCache, "Starting CleanAgedItems")
```

Comment:
I don't think this logging provides particularly useful diagnostic information - I expect it would just be noise during problem analysis.   I'd suggest we remove.